mex/common/wikidata/extract.py

```diff
@@ -54,7 +54,12 @@ def search_organization_by_label(
                 f"IndexError: Error processing results for {item_label}"
             ) from exc
 
-        yield _get_organization_details(wd_item_id)
+        resolved_organizations.append(_get_organization_details(wd_item_id))
+
+    if len(resolved_organizations) == 1:
```

Comment:
not very efficient. we could bail out with `None` as soon as we find a second org in `results` - no need to finish the loop

Reply:
Good point, fixed it in 443c1a2a28dea9bc5394d5e2eeff3320543c0e02

Reply:
this commit changes the interface of this function though. before, if you had 0 results, you'd still get a `None`, now you get a `MExError`

Reply:
What do you think about a simpler solution like this: c555328cf8ee1896705dcbebab1727453fb9b872

Reply:
yup, that works as well.

but: the IndexError can not happen anymore, so you could trash that. it was not covered by tests anyway

```
 Name                                     Stmts   Miss Branch BrPart  Cover   Missing
------------------------------------------------------------------------------------
mex/common/wikidata/extract.py              21      2      4      1    88%   42-43
```